integration-tests/keycloak-authorization/src/main/resources/application.properties

```diff
@@ -1,9 +1,11 @@
+quarkus.keycloak.devservices.create-realm=false
+quarkus.keycloak.devservices.resource-aliases.policies=${project.build.directory}/${project.build.finalName}-policies.jar
```

Comment:
@pedroigor @gsmet FYI, this is how a generated policy jar is attached to the container, this way arbitrary class or file system resources can be mapped to corresponding Keycloak container resources.

This requires the introduction of two Map properties and I was hesitating but then I thought it could male it so much easier for users to customize containers with some extra scripts, resources etc

Reply:
Yeah, it is more flexible. So now you can install any custom provider to Keycloak (or any other resource).